scripts/tasks/src/type-check.ts

```diff
@@ -34,6 +34,8 @@ export function typeCheck() {
   return exec(program)
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30987&commitId=0e55f341319f37f8ff806a7549d20b00ed629c4c&env=prod&clientType=FLUENTUI)


  <details><summary><b>Callout</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Callout.Left top edge.default.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>